src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -98,6 +124,18 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
     setDeploymentState('loading');
     setIsLoading(true);
 
+    // this logic is here to test locally without publishing anything
+    const hostingPlatformVal = {};
+    if (hostingPlatforms.hasOwnProperty(inputValues.platform)) {
+      hostingPlatformVal[inputValues.platform] = { ...inputValues };
+    }
+    saveLocalStorage(localStorageSharableUrl, hostingPlatformVal);
+    const newState = {
+      ...hostingPlatformLocalStorageVal,
+      ...hostingPlatformVal,
+    };
+    setHostingPlatformLocalStorageVal(newState);
```

Comment:
Nit: This part about localStorage can be moved into a separate function.

Reply:
lets discuss about this when you're back 👍